test/verify/check-pages

```diff
@@ -560,14 +560,27 @@ OnCalendar=daily
         b.wait_in_text("#file-autocomplete-widget li:nth-of-type(2) button", "dir1/")
         b.wait_in_text("#file-autocomplete-widget li:nth-of-type(3) button", "file1.txt")
         b.click("#file-autocomplete-widget li:nth-of-type(2) button")
+        b.wait_in_text("#file-autocomplete-widget li:nth-of-type(1) button", "This directory is empty")
+
+        # clear the file completion widget
+        b.click("#demo-file-ac div:first-of-type div:first-of-type button:nth-of-type(1)")
```

Comment:
this selector is funny :) a bit too complicated it seems (but yeah, I've seen some components where it is almost impossible to get to) but also you use both `first-of-type` and `nth-of-type(1)` :)

This is of course just a joke comments, no need to do anything about it.

Reply:
Haha yea I kept digging up to find the nearest clearly defined/readable id and this was the first(-of-type) :grin: